AISKU/src/applicationinsights-web.ts

```diff
@@ -35,7 +35,8 @@ export {
     INotificationManager,
     IProcessTelemetryContext,
     Tags,
-    BaseCore
+    BaseCore,
+    CoreUtils
```

Comment:
Wasn't directly exported but was indirectly available because of the internal usages, so to avoid breaking anyone how had taken a dependency explicitly exporting.